e2e/tests/graph.spec.ts

```
@@ -1,75 +1,258 @@
-            const downloadPath = await download.path();
-            expect(fs.existsSync(downloadPath)).toBe(true);
+            await graph.insertQuery(BATCH_CREATE_PERSONS);
+            await graph.clickRunQuery();
+
+            const searchQuery = `Person ${node}`;
+            await graph.searchForElementInCanvas(searchQuery);
+            await graph.hoverAtCanvasCenter();
+            expect(await graph.getNodeCanvasToolTip()).toBe(searchQuery);
+
+            await apicalls.removeGraph(graphName);
         });
```

ESLint

[error] Function declared in a loop contains unsafe references to variable(s) 'browser', 'browser', 'apicalls'.

(@typescript-eslint/no-loop-func)